src/provider/cloud-element-templates/properties/CustomProperties.js

```diff
@@ -317,7 +319,8 @@ function TextAreaProperty(props) {
   const {
     description,
     editable,
-    label
+    label,
+    feel
```

Comment:
We don't already validate this property via JSON Schema, right? Will we bump the schema + validator in a follow-up PR?

Reply:
Good point!

So within this PR we should also bump https://github.com/bpmn-io/element-templates-validator to a version, which will understand the feel property.

(This would requires us to release https://github.com/bpmn-io/element-templates-validator first of course...)

Reply:
I will take care of the releases tomorrow and bump it here

Reply:
that is my understanding, yes.

Reply:
> Right, we did not bump the schma yet. I could not find a dependency for the JSON schema version, is this done through `@bpmn-io/element-templates-validator`?

Exactly, the validator is also exposing the supported Schema Version (for both schemas).

Reply:
Cf. https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/src/provider/cloud-element-templates/Validator.js#L13

I think it's also good practice to add some test coverage that the validator is understanding a new property correctly, e.g. what we've done previously with the new optional binding: https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/test/spec/provider/cloud-element-templates/Validator.spec.js#L379

Reply:
Done with [401b8e0](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/602/commits/401b8e09c21ed67416ecc89b430344a08db4cb51) and [662cac3](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/602/commits/662cac38f5867c2491c7821fd2ea316908396220) (already squashed).

Validator test was added here: https://github.com/bpmn-io/bpmn-js-properties-panel/blob/662cac38f5867c2491c7821fd2ea316908396220/test/spec/provider/cloud-element-templates/Validator.spec.js#L396-L410